distribution/docs/src/main/resources/content/_quickstart/quickstart-configuring.adoc

```diff
@@ -14,7 +14,8 @@ Set the configurations needed to run ${branding}.
 . Follow the installer prompts for a standard installation.
 .. Click start to begin the setup process.
 .. Configure <<_guest_interceptor,guest claims attributes>> or use defaults.
-... *All users will be automatically granted these authorizations*
+... *All users will be automatically granted these permissions.*
+... *Any data ingested will need to have these attributes to be accessible by Guest users.*
```

Comment:
If a metacard is ingested without any security attributes it will be accessible by guests as well. These attributes represent the highest metacard security attributes a guest can query/ingest. Not sure how to say that succinctly.

Reply:
It's hard to talk about guest attributes without first defining what a Guest is. I think I it worthwhile to repeating (and elaborate on) the explanation from section 4.6.1.:

`Unauthenticated access to a secured DIB system is provided by the Guest user. By default, the Search UI allows guest access as part of the karaf security realm.` and the elaboration:

`Because DIB does not know the identify of a Guest user, it cannot assign security attributes to the Guest. The administrator must configure the attributes and values (i.e. the "claims") to be assigned to Guests. The Guest Claims become the default attributes for every user, both authenticated and unauthenticated.  Every user is automatically imbued with all Guest Claims.
WARNING: Content ingested by DIB is marked with the System High security settings, unless configured otherwise. If Guest Claims are less restrictive than System High settings (which is typical), then Guests will be able to ingest contents, but not retrieve it.`

Caveat Emptor: This is to the best of my understanding. If it is wrong, @clockard  please reply.

Reply:
That's pretty succinct actually

Reply:
Adding your elaboration.

Reply:
Two things:
1. My understanding is that a user cannot ingest content if, on ingest, it would get markings that are "higher" (not sure what word to use here) than the user's claims (which, at a minimum, are the guest claims but could be higher)
2. I don't think this is the right place to be talking about the System High security settings. I think this explanation should just be in relation to the metacard/content's markings.

So I think the Warning should just be one sentence like`If Guest Claims are less restrictive than the content's markings, then guests will not be able to ingest or retrieve that content`.